.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on: [push, pull_request]
 
 jobs:
   checker:
+  SHM:
```

Comment:
Will this work correctly? You can specify two jobs with different names

Reply:
Yes, this is problematic. Merged unsuccessfully. Do not mix jobs.